### DISCUSSION TITLE
5주차 과제 - Creative Painter or Typerwriter + Data Viz Idea

### DISCUSSION THREAD

**Discussion by youngsangcho** [youngsangcho/generativedesign2_2021#17] [Show and tell]
#과제 1
강의 샘플코드의 painter / typewriter 예제를 응용 변형하여 나만의 painter / typewriter 만들기.
혹은 다른 맥락에서 기능만을 이용해도 좋음. 

+ 배열 반드시 사용
+ painter / typewriter에 제목 붙이기(성격 혹은 의미 부여하기)
+ motion, randomness 필수.
+ (추가 점수) 지우개 등 특별한 기능

+ 1개 이상의 스크린샷 이미지
+ 스크린 동영상 캡쳐하여 유튜브 업로드 유튜브 embed
+ 간단한 설명 (왜 / 어떻게)
+ p5js editor 의 File - Share - Present 링크
+ p5js editor 의 File - Share - Edit 링크
+ 코드 복사 붙여넣기

- - - - - - - -
#과제2
향후 데이터 시각화 과제에 사용할 아이디어 2 ~ 3개.
위 과제와 다른 별도의 블로그 글 생성.

+ '나' 혹은 '내 주변'에 관한 데이터.
+ 내가 (짧은 시간 안에) 모을 수 있거나, 이미 쌓여있는 데이터. 시각화할 의미가 있는 데이터.
+ 간략하게 글로 정리.
+ 레퍼런스 작업? (optional)

**Comment by eunjinhong** [comment]
홍은진
과제1 https://blog.naver.com/tnekf1012/222540873909
과제2 https://blog.naver.com/tnekf1012/222540909164

**Comment by KimHyunJuhg** [comment]
김현정
https://blog.naver.com/khj317888/222540841501

**Reply by KimHyunJuhg** [reply]
과제 1,2 모두 같은 글에 구분해서 적었습니다!

**Comment by SeyounJung01** [comment]
정세윤
https://blog.naver.com/seyounjj2001/222540985285
https://blog.naver.com/seyounjj2001/222540995069

**Comment by imyeeunok** [comment]
https://blog.naver.com/010312kye/222540992443

**Reply by youngsangcho** [reply]
블로그 포스트의 질문에 대한 답변 예제 코드
https://editor.p5js.org/youngsangcho/sketches/L7mc19oj9

**Comment by Joeun410** [comment]
김조은
https://blog.naver.com/PostView.naver?blogId=joanne3456&Redirect=View&logNo=222541070332&categoryNo=12&isAfterWrite=true&isMrblogPost=false&isHappyBeanLeverage=true&contentLength=46018

**Comment by jwejwepark** [comment]
https://blog.naver.com/glqkfl0726/222541365441
